locale/sv/LC_MESSAGES/BingEarthWallpaper.po

```diff
@@ -0,0 +1,172 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-05-30 15:02+1000\n"
+"PO-Revision-Date: 2020-07-28 20:01+0200\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.3.1\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: sv\n"
+
+#: Settings.ui.h:1
+msgid "Hide the indicator"
+msgstr "Göm indikatorn"
+
+#: Settings.ui.h:2
+msgid "Indicator icon"
+msgstr "Indikatorikon"
+
+#: Settings.ui.h:3 extension.js:164
+msgid "Set background image"
+msgstr "Välj backgrundsbild"
+
+#: Settings.ui.h:4 extension.js:165
+msgid "Set lock screen image"
+msgstr "Välj låsskärmsbild"
+
+#: Settings.ui.h:5
+msgid "Download folder:"
+msgstr "Nedladdningsmapp:"
+
+#: Settings.ui.h:6
+msgid "Bing Wallpaper pictures folder"
+msgstr "Bing Wallpaper nedladdningsmapp"
+
+#: Settings.ui.h:7
+msgid "Delete previously downloaded wallpapers:"
+msgstr "Ta bort tidigare nedladdade bakgrundsbilder:"
+
+#: Settings.ui.h:8
+msgid "Days to store wallpapers before deleting:"
+msgstr "Antal dagar innan bilder tas bort:"
+
+# There is no translations of "locale", in this context that I'm aware of. I would say that you use the english words in swedish aswell
```

Comment:
In the Bing API they use the terminology "market", as in the "Swedish market". The images are sometimes different, so it not just a language thing. This is probably most apparent for national days, or perhaps other subjects that might be sensitive. I use locale as it's sometimes used to describe translated programs text to a local language, as well as formatting things like number or date formats. In this case you can think of locale as being roughly equivalent to the regional language, but it has some implications for the Bing images too (which I talk about below)

Reply:
I borrowed a PC with Windows 10 to see how Microsoft translates it they seem to just translate locale to "språk" (language). For example "System locale" is translated to "Systemspråk".  Using the same logic a translation could be "Bing språkregion". However I think a better translation would be "Bing lands- och språkregion" which translates to "country- and language region".

Let me know what you think and if it fits the UI

Reply:
Okay I thought more about the translation since it bothered me. I did some googling and according to this [site](https://sv.bab.la/lexikon/engelsk-svensk/locale) _locale_ can also be translated to _språkvariant_. I think that would be the best translation.

Reply:
That sounds sensible :)